service/migration_manager.cc

```diff
@@ -915,13 +915,13 @@ future<> migration_manager::announce_without_raft(std::vector<mutation> schema,
     try {
         using namespace std::placeholders;
         auto all_live = _gossiper.get_live_members();
-        auto live_members = all_live | boost::adaptors::filtered([this, my_address = _messaging.broadcast_address()] (const gms::inet_address& endpoint) {
+        auto live_members = all_live | std::views::filter([this, my_address = _messaging.broadcast_address()] (const gms::inet_address& endpoint) {
             // only push schema to nodes with known and equal versions
             return endpoint != my_address &&
                 _messaging.knows_version(endpoint) &&
                 _messaging.get_raw_version(endpoint) == netw::messaging_service::current_version;
         });
-        co_await coroutine::parallel_for_each(live_members.begin(), live_members.end(),
+        co_await coroutine::parallel_for_each(live_members,
```

Comment:
yes, it is better - it would be appreciated if other changes in this PR would be mentioned also in the PR's description. E.g. "changed from iterators interface to ranges interface when applicable".